src/main/java/net/pms/dlna/DVDISOFile.java

```diff
@@ -90,7 +109,7 @@ protected void resolveOnce() {
 			"null",
 			"-dvd-device",
 			ProcessUtil.getShortFileNameIfWideChars(file.getAbsolutePath()),
-			"dvd://1"
+			"dvd://"
```

Comment:
@Sami32 This isn't used to *play* a DVD, it's used when parsing it. Therefore selecting a track isn't relevant. I've seen that MPlayer picks a track anyway, so it didn't really "help". What I wanted was not to get information about a particular track at all, only the list of tracks on the disc.